app/code/Magento/InventoryBundleIndexer/Test/Integration/SourceItemIndexerTest.php

```diff
@@ -0,0 +1,165 @@
+     */
+    public function testReindexWithAllChildrenInStock()
+    {
+        $bundleStockItemData = $this->getStockItemData->execute('bundle-product-eu-website', 10);
+
+        self::assertEquals(true, (bool)$bundleStockItemData['is_salable']);
+    }
+
+    /**
+     * @magentoDataFixture ../../../../app/code/Magento/InventorySalesApi/Test/_files/websites_with_stores.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/products.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryBundleIndexer/Test/_files/bundle_product_eu_website.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/sources.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/stocks.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/stock_source_links.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryBundleIndexer/Test/_files/source_items_bundle.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventorySalesApi/Test/_files/stock_website_sales_channels.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryIndexer/Test/_files/reindex_inventory.php
+     *
+     * @return void
+     */
+    public function testReindexWithOneChildOutOfStock()
+    {
+        $this->makeChildrenOutOfStock(1);
+        $bundleStockItemData = $this->getStockItemData->execute('bundle-product-eu-website', 10);
+
+        self::assertEquals(true, (bool)$bundleStockItemData['is_salable']);
+    }
+
+    /**
+     * @magentoDataFixture ../../../../app/code/Magento/InventorySalesApi/Test/_files/websites_with_stores.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/products.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryBundleIndexer/Test/_files/bundle_product_eu_website.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/sources.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/stocks.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryApi/Test/_files/stock_source_links.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryBundleIndexer/Test/_files/source_items_bundle.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventorySalesApi/Test/_files/stock_website_sales_channels.php
+     * @magentoDataFixture ../../../../app/code/Magento/InventoryIndexer/Test/_files/reindex_inventory.php
+     *
+     * @return void
+     */
+    public function testReindexWithAllChildrenOutOfStock()
+    {
+        $this->makeChildrenOutOfStock(3);
+        $bundleStockItemData = $this->getStockItemData->execute('bundle-product-eu-website', 10);
+
+        self::assertEquals(false, (bool)$bundleStockItemData['is_salable']);
+    }
+
+    /**
+     * @param int $qty
+     * @return void
+     */
+    private function makeChildrenOutOfStock(int $qty)
```

Comment:
`$qty` is bad name for variable
Generally, qty is mean quantity of product (especially if we work with inventory)

Reply:
Fixed.